Userland/Services/LookupServer/DNSPacket.cpp

```diff
@@ -153,6 +153,12 @@ Optional<DNSPacket> DNSPacket::from_raw_packet(const u8* raw_data, size_t raw_si
             break;
         }
         case DNSRecordType::A:
+            // Fall through
+        case DNSRecordType::TXT:
+            // Fall through
+        case DNSRecordType::AAAA:
+            // Fall through
+        case DNSRecordType::SRV:
```

Comment:
This commit no longer does what it says it does (a rebase gone wrong?)

Reply:
Yup, that happened during a rebase. Should be OK now.